ui/webui/README.rst

```diff
@@ -8,6 +8,8 @@ Make sure you have `npm` available (usually from your distribution package).
 These commands check out the source and build it into the `dist/` directory::
 
     git clone https://github.com/rhinstaller/anaconda.git
+    cd anaconda
```

Comment:
I would prefer that we point then on the CONTRIBUTING GUIDE on 'how to run make commands'  

https://github.com/rhinstaller/anaconda/blob/master/CONTRIBUTING.rst#how-to-run-make-commands

So that they have the alternative of using toolbox, and correctly set up deps and so on.